README.md

```diff
@@ -1,2 +1,6 @@
 # imu
 Repo for launching IMU driver on AV
+
+## Info
+
+Lord GX5-3DM-15 IMU
```

Comment:
Would be good if we can explain the docker scripts and the use of them (we can copy and paste this description across other AV repos).

As a context: For me when I see `build.sh` and `run.sh` along with a dockerfile, I immediately think that `build.sh` is for literally only building the docker and `run.sh` is to run the built image (and potentially bash CMD accepted as args). Instead these scripts always build the docker image up to a certain target - either `build` and `dev`.

Reply:
I have added an #Usage section